packages/core/src/navigation/utils/index.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directions, setAttributes, setOrRemoveAttribute } from '@cds/core/internal';
+import { CdsNavigationGroup, CdsNavigationItem, CdsNavigationStart } from '../index.js';
+
+export const NAVIGATION_ITEM_TEXT = 'navigation-item-text';
+export const DEFAULT_NAVIGATION_LAYOUT = 'vertical';
+
+export type FocusableElement = CdsNavigationItem | CdsNavigationStart;
+
+export function expandedGroupEvent(event: KeyboardEvent): boolean {
+  const element = event.target as HTMLElement;
+  return element?.parentElement?.tagName === 'CDS-NAVIGATION-GROUP' && element?.parentElement?.hasAttribute('expanded');
+}
+
+// Something wrong here
+export function expandedNavigation(event: KeyboardEvent): boolean {
+  const element = event.target as HTMLElement;
+  return element?.parentElement?.tagName === 'CDS-NAVIGATION' && element?.parentElement?.hasAttribute('expanded');
```

Comment:
Same comment here as above... I wasn't expecting this to return to Boolean like this.

Reply:
I updated all of the util fn names to follow the `isSomethingSomethingEvent` pattern.